src/ceph-volume/ceph_volume/util/arg_validators.py

```diff
@@ -51,13 +52,48 @@ def _is_valid_device(self):
         return self._device
 
 
+def get_osd_ids_up():
+    bootstrap_keyring = '/var/lib/ceph/bootstrap-osd/%s.keyring' % conf.cluster
```

Comment:
I know for running through the ceph-volume containers that cephadm deploys this keyring is being handled. For users just running this directly will they need to manually setup this keyring here? If so, perhaps we want some way to skip this check to make it a bit easier for them to use?

Reply:
I don't think we need to conditionally skip this, ceph-volume has always done such calls when preparing the device, for instance, see [1].

[1] https://github.com/ceph/ceph/blob/main/src/ceph-volume/ceph_volume/util/prepare.py#L148-L211